_non-cosmos/ethereum/assetlist.json

```diff
@@ -1018,6 +1018,35 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/pstake.svg"
       },
       "coingecko_id": "pstake-finance"
+    },
+    {
+      "description": "The OCC token acts as the fuel for the OccamRazer launchpad and it is indirectly used as well on the DAO governance layer.",
+      "type_asset": "erc20",
+      "address": "0x2F109021aFe75B949429fe30523Ee7C0D5B27207",
+      "denom_units": [
+        {
+          "denom": "0x2F109021aFe75B949429fe30523Ee7C0D5B27207",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "occ",
+          "exponent": 18
+        }
+      ],
+      "base": "0x2F109021aFe75B949429fe30523Ee7C0D5B27207",
+      "name": "OccamFi",
+      "display": "occ",
+      "symbol": "OCC",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/gravitybridge/images/occamfi.png"
+      },
+      "coingecko_id": "occamfi",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/gravitybridge/images/occamfi.png"
```

Comment:
It makes more sense to store the image under ethereum/images/ instead of in gravitybridge/images/

Reply:
fixed